cmd/compose/compose.go

```diff
@@ -601,42 +599,6 @@ func RootCommand(dockerCli command.Cli, backend Backend) *cobra.Command { //noli
 	return c
 }
 
-// If user has a local .env file, load it as os.environment so it can be used to set COMPOSE_ variables
-// This also allows to override values set by the default .env in a compose project when ran from a distinct folder
-func setEnvWithLocalDotEnv(prjOpts *ProjectOptions) error {
```

Comment:
Question: why are we removing this?

Reply:
this is the way we used to load PWD/.env before we know the actual project directory. This logic is now covered by toProjectOptions